frontend/src/routines/user.ts

```diff
@@ -2,5 +2,6 @@ import { createRoutine } from 'redux-saga-routines';
 
 const addNewUserRoutine = createRoutine('ADD_NEW_USER');
 const fetchUserRoutine = createRoutine('FETCH_USER');
+const editProfile = createRoutine('EDIT_PROFILE');
 
-export { addNewUserRoutine, fetchUserRoutine };
+export { addNewUserRoutine, fetchUserRoutine, editProfile };
```

Comment:
```suggestion
export { addNewUserRoutine, fetchUserRoutine, editProfileRoutine };
```

please follow one style